docs/dc-apps-performance-toolkit-user-guide-bitbucket.md

```diff
@@ -0,0 +1,577 @@
+---
+title: "Data Center App Performance Toolkit User Guide For Bitbucket"
+platform: platform
+product: marketplace
+category: devguide
+subcategory: build
+date: "2020-02-13"
+---
+# Data Center App Performance Toolkit User Guide For Bitbucket
+
+To use the Data Center App Performance Toolkit, you'll need to first clone its repo.
+
+``` bash
+git clone git@github.com:atlassian/dc-app-performance-toolkit.git
+```
+
+Follow installation instructions described in the `dc-app-performance-toolkit/README.md` file.
+
+If you need performance testing results at a production level, follow instructions in this chapter to set up Bitbucket Data Center with the corresponding dataset.
+
+For spiking, testing, or developing, your local Bitbucket instance would work well. Thus, you can skip this chapter and proceed with [Testing scenarios](/platform/marketplace/dc-apps-performance-toolkit-user-guide-bitbucket/#testing-scenarios). Still, script adjustments for your local dataset may be required.
+
+## Setting up Bitbucket Data Center
+
+We recommend that you use the [AWS Quick Start for Bitbucket Data Center](https://aws.amazon.com/quickstart/architecture/bitbucket/) to deploy a Bitbucket Data Center testing environment. This Quick Start will allow you to deploy Bitbucket Data Center with a new [Atlassian Standard Infrastructure](https://aws.amazon.com/quickstart/architecture/atlassian-standard-infrastructure/) (ASI) or into an existing one.
+
+The ASI is a Virtual Private Cloud (VPC) consisting of subnets, NAT gateways, security groups, bastion hosts, and other infrastructure components required by all Atlassian applications, and then deploys Jira into this new VPC. Deploying Jira with a new ASI takes around 50 minutes. With an existing one, it'll take around 30 minutes.
+
+### Using the AWS Quick Start for Bitbucket
+
+If you are a new user, perform an end-to-end deployment. This involves deploying Bitbucket into a _new_ ASI.
+
+If you have already deployed the ASI separately by using the [ASI Quick Start](https://aws.amazon.com/quickstart/architecture/atlassian-standard-infrastructure/)ASI Quick Start or by deploying another Atlassian product (Jira, Bitbucket, or Confluence Data Center), deploy Bitbucket into your existing ASI.
```

Comment:
Delete please the second one "ASI Quick Start"
![image](https://user-images.githubusercontent.com/30836532/75034085-123ca700-54b5-11ea-890a-d3d81916b6cb.png)